.github/workflows/semconvgen-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Semantic Convention Generator Run Tests
+on:
+  push:
+    tags: [ '**' ]
```

Comment:
Do we want to run this on tags? If so, is there a way to skip publishing if the test workflow fails?

Reply:
I thought maybe that `jobs.{job_id}.needs` was the solution here, but I guess not -- the build job just didn't run at all. I have one more thing to try.

Reply:
That did the trick. The two jobs run in sequence now, which may cause the builds to take longer than they otherwise would, but this seems like a reasonable trade off in a project that doesn't get built that often.